src/Entity/ProcurationRequest.php

```diff
@@ -352,6 +352,13 @@ class ProcurationRequest
      */
     public $reachable = false;
 
+    /**
+     * @ORM\Column(type="boolean", options={"default": false})
+     *
+     * @Assert\Type("bool")
+     */
+    private bool $enabled = false;
```

Comment:
with typehint `bool $enabled` we can avoid using `Assert\Type`, wdyt?

```suggestion
    /**
     * @ORM\Column(type="boolean", options={"default": false})
     */
    private bool $enabled = false;
```